astropy/io/votable/connect.py

```diff
@@ -225,10 +226,11 @@ def write_table_votable_parquet(input, output, column_metadata, *, overwrite=Fal
     file is f"{output}.parquet".
     """
     # First save the PARQUET file.
-    parquet_filename = f"{output}.parquet"
-    path_type = f"file:{'//' if os.path.isabs(parquet_filename) else ''}"
+    output_path = Path(output)
+    parquet_filename = Path(f"{output}.parquet")
```

Comment:
```suggestion
    parquet_filename = output_path.with_suffix("parquet")
```

Reply:
unfortunately not equivalent, unless we can trust that `output` never comes with a suffix of its own, which I don't know that we can.